utils/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "utils"
+publish = false
```

Comment:
This will need to be published since it's used in the build, will have to pick a more unique name for it

Reply:
`hydro-build-utils` or something